tests/fixtures/sort/keys_open_ended.expected.debug

```diff
@@ -0,0 +1,18 @@
+gg aa cc
+    ____
+________
+dd aa ff
+    ____
+________
+aa bb cc
+    ____
+________
+èè éé èè
+    ____
+________
+👩‍🔬 👩‍🔬 👩‍🔬
+       _______
```

Comment:
While this looks off in the browser, it should look correct in a terminal. 👩‍🔬 is actually 👩 +zero_width_joiner+🔬 and is displayed as two characters in my terminal.
This is how it looks in my terminal:
![Bildschirmfoto von 2021-04-24 10-57-28](https://user-images.githubusercontent.com/44171190/115953449-e7504780-a4eb-11eb-95ad-0e4de60d44ae.png)

Reply:
the best kind of test :)